packages/databricks-sdk-js/src/config/oauth/Issuer.ts

```diff
@@ -0,0 +1,76 @@
+import {Config} from "../Config";
+import {Client, ClientOptions} from "./Client";
+import fetch from "node-fetch-commonjs";
+
+export class Issuer {
```

Comment:
we call this `OidcEndpoints`, happy to rename it in the other SDKs

Reply:
I can rename, no problem